### DISCUSSION TITLE
2D point representation- size

### DISCUSSION THREAD

**Discussion by Quides5** [BaPSF/bapsf_motion#6] [Ideas]
I've implemented the representation of points on the 2D canvas projection.
Currently the points are drawn as individual pixels, but I can also make it so that they are displayed as circles whose size changes dynamically as per the resolution of the grid. Which style would we prefer?

**Comment by rocco8773** [comment]
Without seeing how it looks, my intuition says a single pixel will be too small for the human.  Thus, a dynamically changing circle will likely be the best solution.
